db/schema.rb

```diff
@@ -0,0 +1,63 @@
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# Note that this schema.rb definition is the authoritative source for your
+# database schema. If you need to create the application database on another
+# system, you should be using db:schema:load, not running all the migrations
+# from scratch. The latter is a flawed and unsustainable approach (the more migrations
+# you'll amass, the slower it'll run and the greater likelihood for issues).
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema.define(version: 2020_09_02_173624) do
+
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
+
+  create_table "items", force: :cascade do |t|
+    t.string "name"
+    t.integer "price"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
+  create_table "shipment_items", force: :cascade do |t|
+    t.bigint "item_id"
+    t.bigint "shipment_id"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["item_id"], name: "index_shipment_items_on_item_id"
+    t.index ["shipment_id"], name: "index_shipment_items_on_shipment_id"
+  end
+
+  create_table "shipments", force: :cascade do |t|
+    t.integer "status"
+    t.string "delivery_date"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.bigint "subscription_id"
+    t.index ["subscription_id"], name: "index_shipments_on_subscription_id"
+  end
+
+  create_table "subscriptions", force: :cascade do |t|
+    t.string "delivery_day"
```

Comment:
This should be called recurring day, will change in later iterations. It's meant to be the day that the subscription would be triggered to be refilled depending on the typeof subscription a user had.